ui/app/templates/vault/cluster/access/method/item/list.hbs

```diff
@@ -1,6 +1,5 @@
 <GeneratedItemList
   @model={{this.model}}
-  @method={{this.method}}
```

Comment:
Method is not being used?

Reply:
Nope, did a double/triple check. It was never in either the js or hbs file. Good catch though.